dags/nci_ard/nci_ls_ard.py

```diff
@@ -47,10 +48,12 @@
     schedule_interval = None
     params["run_ard_arg"] = ""
     params["index_arg"] = ""  # No indexing
+    current_userid = "gy5636"
```

Comment:
```suggestion
    # To use when testing changes
    # current_userid = "gy5636"
```
For production/the master branch, if this block of code is executed it should write to the production log area. Noting the block of code shouldn't run in production, so this is more of a fail safe issue.

Reply:
hmmm...
if we run this in production, it would be when the following lines are satisfied, right?

`if (
    environ.get("AIRFLOW__WEBSERVER__BASE_URL")
    == "https://airflow.sandbox.dea.ga.gov.au"
):`
If that is not the case, i think we should decide on a way that signifies we are running the source code in the production environment.

 Commenting lines 51 and 52 above would only mean that we are committing code that will write to the production log path regardless. That's a bit of a danger.